src/radians.rs

```diff
@@ -23,7 +23,7 @@ impl Radians {
     /// let rotation = Radians::try_new(PI).expect("Value was outside the range [0.0; 2π)");
     /// ```
     pub fn try_new(value: f64) -> Result<Self, RadiansError> {
-        if value >= 0.0 && value < 2.0 * PI {
+        if (0.0..TAU).contains(&value) {
```

Comment:
Could you store this range in a constant?

Reply:
ad36a45313dfb6ea3f23aea9be959d1a9dfc975b